core/src/fetcher.rs

```diff
@@ -204,6 +207,7 @@ pub async fn fetch_chain_data(
             mev_commission_bps,
             mev_revenue_lamports,
             running_jito,
+            running_bam: matches!(client_type, Some(ClientType::Bam)),
```

Comment:
Nit; would prefer a `let is_bam_client ...` just like we have `let is_jito_client` instead of the inline eval here.